mkdocs.yml

```diff
@@ -1,204 +1,225 @@
+        - V3 Migration: microservices/application/V3Migration.md
+      - SDKs:
+        - microservices/application/ApplicationFunctionsSDK.md
+        - microservices/device/sdk/Ch-DeviceSDK.md
+      - Configuration & Registry:
+        - microservices/configuration/ConfigurationAndRegistry.md
+        - microservices/configuration/CommonConfiguration.md
+        - microservices/configuration/CommonCommandLineOptions.md
+        - microservices/configuration/CommonEnvironmentVariables.md
+        - V3 Migration: microservices/configuration/V3MigrationCommonConfig.md
+      - microservices/general/index.md
+      - microservices/general/messagebus.md
+    - Security:
+      - Security Services: security/Ch-Security.md
+      - security/Ch-SecretStore.md
+      - security/Ch-Authenticating.md
+      - security/Ch-SecretProviderApi.md
+      - security/Ch-APIGateway.md
+      - security/Ch-Secure-MessageBus.md
+      - security/Ch-Configuring-Add-On-Services.md
+      - security/Ch-Secure-Consul.md
+      - security/Ch-SecurityIssues.md
+      - security/secrets-config.md
+      - security/secrets-config-proxy.md
+      - security/SeedingServiceSecrets.md
+      - security/Ch-CORS-Settings.md
+      - security/Ch-DelayedStartServices.md
+      - security/Ch-RemoteDeviceServices.md
+      - V3 Migration: security/V3Migration.md
+      - Threat Models:
+        - threat-models/secret-store/README.md
+        - threat-models/stride-model/EdgeX-STRIDE.md
+        - threat-models/stride-model/EdgeXFoundryThreatReportV2.2.md
+    - Tutorials:
+      - examples/Ch-ExamplesVirtualDeviceService.md
+      - examples/Ch-ExamplesSendingAndConsumingBinary.md
+      - examples/Ch-CommandingDeviceThroughRulesEngine.md
+      - examples/Ch-ExamplesModbusdatatypeconversion.md
+      - examples/Ch-OSImageWithEdgeX.md
+      - Adding a Device:
+        - examples/Ch-ExamplesAddingModbusDevice.md
+        - examples/Ch-ExamplesAddingSNMPDevice.md
+        - examples/Ch-ExamplesAddingMQTTDevice.md
+      - Walkthrough:
+        - walk-through/Ch-Walkthrough.md
+        - walk-through/Ch-WalkthroughSetup.md
+        - walk-through/Ch-WalkthroughUseCase.md
+        - walk-through/Ch-WalkthroughDeviceProfile.md
+        - walk-through/Ch-WalkthroughDeviceService.md
+        - walk-through/Ch-WalkthroughProvision.md
+        - walk-through/Ch-WalkthroughCommands.md
+        - walk-through/Ch-WalkthroughReading.md
+        - walk-through/Ch-WalkthroughExporting.md
+    - APIs: 
+      - api/Ch-APIIntroduction.md
+      - api/core/Ch-APICoreCommand.md
+      - api/core/Ch-APICoreData.md
+      - api/core/Ch-APICoreMetadata.md
+      - api/core/Ch-APICoreConfigurationAndRegistry.md
+      - api/support/Ch-APISupportNotifications.md
+      - api/support/Ch-APISupportScheduler.md
+      - api/devices/Ch-APIDeviceSDK.md
+      - api/applications/Ch-APIAppFunctionsSDK.md
+      - api/applications/Ch-APIRulesEngine.md  
+  - Migration:
```

Comment:
This is the unfortunate thing with cross referenced links.  We can reference things under other menu drop downs, but if it is a reference, it takes you to the original.  Options include: take migration out of the top menu bar and have it on the left nav, take it out of the left nav (under all the different areas), or replicate content.

Reply:
Yep, that is why I think Migration and others should be in the LHS menu, not the menu bar.

Reply:
Moved Migration, Developer and Examples down to LHS.  Kept Home, Reference and Use Case & Design because I think these are different from the system documentation and deserve their own section.  But I can be persuaded by general opinion of the community to also move these down. 

Migration docs are now referenced in 2 different areas (under Migration and in their own section).  This can still lead to a jarring experience if you are in one of the other sections, it will take you to the Migration area in the LHS menu bar.